### DISCUSSION TITLE
DIfferences of two files in a new file.

### DISCUSSION THREAD

**Discussion by Janeyaga** [WinMerge/winmerge#1610] [Q&A]
Hi,

I have to compare two large files . And I would like to see the result in a new file as it is difficult to scroll down.

is it possible to have the result in new file ?

Regards
Jane

**Comment by sdottaka** [comment]
I may not understand exactly what you want to do, but first, if there are few differences, you can select the View → Diff Context → 0 Lines menu item to only show the differences, which can reduce the amount of scrolling you need to do. Then, you can select the Tools → Generate Patch or Generate Report menu item to output the differences to a file.
